test/sql-tap/gh-2549-many-columns.test.lua

```diff
@@ -37,7 +37,8 @@ test:do_catchsql_test(
 	"columns-1.2",
 	fail_statement, {
 		-- <columns-1.2>
-		1, "Failed to create space 'T2': space column count 2001 exceeds the limit (2000)"
+        1, "Failed to create space 't2': space column count 2001 exceeds the "..
+        "limit (2000)"
```

Comment:
Typo: I suggest to leave the original indent level.

Reply:
See above.